### ISSUE TITLE
nodemon crashes on `yarn build:watch`

### ISSUE THREAD

**Issue by Tobbe** [redwoodjs/redwood#678]
redwood@d51ade08118c17459cebcdb496197ea52485364a
windows 10
GitBash (mintty 3.1.6)
yarn v1.22.4

```
$ yarn build:watch
yarn run v1.22.4
$ lerna run build:watch --parallel
lerna notice cli v3.22.0
lerna info Executing command in 11 packages: "yarn run build:watch"
@redwoodjs/core: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/cli: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/auth: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/dev-server: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
create-redwood-app: $ nodemon --ignore dist --exec 'yarn build'
@redwoodjs/api: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --ignore importAll.macro.js --exec 'yarn build'
@redwoodjs/internal: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/eslint-plugin-redwood: $ nodemon --ignore dist --exec 'yarn build'
@redwoodjs/router: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/cli: [nodemon] 2.0.4
@redwoodjs/cli: [nodemon] to restart at any time, enter `rs`
@redwoodjs/cli: [nodemon] watching path(s): src\**\*
@redwoodjs/cli: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/cli: [nodemon] starting `'yarn build'`
@redwoodjs/web: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/cli: ''yarn' is not recognized as an internal or external command,
@redwoodjs/cli: operable program or batch file.
@redwoodjs/testing: $ nodemon --watch src --ext 'js,ts,tsx' --ignore dist --exec 'yarn build'
@redwoodjs/cli: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/auth: [nodemon] 2.0.4
@redwoodjs/core: [nodemon] 2.0.4
@redwoodjs/auth: [nodemon] to restart at any time, enter `rs`
@redwoodjs/auth: [nodemon] watching path(s): src\**\*
@redwoodjs/core: [nodemon] to restart at any time, enter `rs`
@redwoodjs/auth: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/auth: [nodemon] starting `'yarn build'`
create-redwood-app: [nodemon] 2.0.4
@redwoodjs/core: [nodemon] watching path(s): src\**\*
@redwoodjs/core: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/core: [nodemon] starting `'yarn build'`
create-redwood-app: [nodemon] to restart at any time, enter `rs`
create-redwood-app: [nodemon] watching path(s): *.*
create-redwood-app: [nodemon] watching extensions: js,mjs,json
create-redwood-app: [nodemon] starting `'yarn build'`
@redwoodjs/dev-server: [nodemon] 2.0.4
@redwoodjs/dev-server: [nodemon] to restart at any time, enter `rs`
@redwoodjs/dev-server: [nodemon] watching path(s): src\**\*
@redwoodjs/dev-server: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/dev-server: [nodemon] starting `'yarn build'`
create-redwood-app: ''yarn' is not recognized as an internal or external command,
create-redwood-app: operable program or batch file.
@redwoodjs/auth: ''yarn' is not recognized as an internal or external command,
@redwoodjs/auth: operable program or batch file.
@redwoodjs/core: ''yarn' is not recognized as an internal or external command,
@redwoodjs/core: operable program or batch file.
@redwoodjs/dev-server: ''yarn' is not recognized as an internal or external command,
@redwoodjs/dev-server: operable program or batch file.
create-redwood-app: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/core: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/dev-server: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/auth: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/api: [nodemon] 2.0.4
@redwoodjs/api: [nodemon] to restart at any time, enter `rs`
@redwoodjs/api: [nodemon] watching path(s): src\**\*
@redwoodjs/api: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/api: [nodemon] starting `'yarn build'`
@redwoodjs/api: ''yarn' is not recognized as an internal or external command,
@redwoodjs/api: operable program or batch file.
@redwoodjs/internal: [nodemon] 2.0.4
@redwoodjs/internal: [nodemon] to restart at any time, enter `rs`
@redwoodjs/internal: [nodemon] watching path(s): src\**\*
@redwoodjs/internal: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/internal: [nodemon] starting `'yarn build'`
@redwoodjs/internal: ''yarn' is not recognized as an internal or external command,
@redwoodjs/internal: operable program or batch file.
@redwoodjs/api: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/internal: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/eslint-plugin-redwood: [nodemon] 2.0.4
@redwoodjs/eslint-plugin-redwood: [nodemon] to restart at any time, enter `rs`
@redwoodjs/eslint-plugin-redwood: [nodemon] watching path(s): *.*
@redwoodjs/eslint-plugin-redwood: [nodemon] watching extensions: js,mjs,json
@redwoodjs/eslint-plugin-redwood: [nodemon] starting `'yarn build'`
@redwoodjs/eslint-plugin-redwood: ''yarn' is not recognized as an internal or external command,
@redwoodjs/eslint-plugin-redwood: operable program or batch file.
@redwoodjs/eslint-plugin-redwood: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/router: [nodemon] 2.0.4
@redwoodjs/router: [nodemon] to restart at any time, enter `rs`
@redwoodjs/router: [nodemon] watching path(s): src\**\*
@redwoodjs/router: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/router: [nodemon] starting `'yarn build'`
@redwoodjs/web: [nodemon] 2.0.4
@redwoodjs/web: [nodemon] to restart at any time, enter `rs`
@redwoodjs/web: [nodemon] watching path(s): src\**\*
@redwoodjs/web: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/web: [nodemon] starting `'yarn build'`
@redwoodjs/router: ''yarn' is not recognized as an internal or external command,
@redwoodjs/router: operable program or batch file.
@redwoodjs/web: ''yarn' is not recognized as an internal or external command,
@redwoodjs/web: operable program or batch file.
@redwoodjs/web: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/router: [nodemon] app crashed - waiting for file changes before starting...
@redwoodjs/testing: [nodemon] 2.0.4
@redwoodjs/testing: [nodemon] to restart at any time, enter `rs`
@redwoodjs/testing: [nodemon] watching path(s): src\**\*
@redwoodjs/testing: [nodemon] watching extensions: 'js,ts,tsx'
@redwoodjs/testing: [nodemon] starting `'yarn build'`
@redwoodjs/testing: ''yarn' is not recognized as an internal or external command,
@redwoodjs/testing: operable program or batch file.
@redwoodjs/testing: [nodemon] app crashed - waiting for file changes before starting...
```

At this point I kill it with Ctrl+C

Lots of "[nodemon] app crashed - waiting for file changes before starting..." and "''yarn' is not recognized as an internal or external command," above.

Running `yarn build` works just fine though.

**Comment by thedavidprice** [comment]
I had this happen to me last week as well. Did not do any diagnosing yet, but I wanted to confirm this is not just a Windows issue.

**Comment by Tobbe** [comment]
Look at this output from the logs above
```
@redwoodjs/api: ''yarn' is not recognized as an internal or external command,
```
Notice the extra ' in front of 'yarn'? Windows don't like the single quotes... So for me the solution is to switch to double quotes. PR incoming!

> I had this happen to me last week as well. Did not do any diagnosing yet, but I wanted to confirm this is not just a Windows issue.
@thedavidprice Not sure what issue you ran in to as you're not on Windows. Maybe open a new issue if it happens to you again?

**Comment by thedavidprice** [comment]
@Tobbe you know, now that you found this fix it could have very well been when I was working on Windows -- the crash stuck with me but the OS did not 😆

Definitely proceed forward here as is. And will loop back when/if needed. Thanks!
